### PR TITLE
Add UMD tests.

### LINKED ISSUE #9: Feature: UMD Wrapper + Tests
Tasks

* [x] Add UMD wrapper
* [x] Likely want to keep `window._lload` in all scenarios for joint "inline + built" bootstrap scenario.
* [ ] TEST: webpack-built frontend
* [ ] TEST: RequireJS frontend

### PR DESCRIPTION
**Pull request by ryan-roemer**
Fixes #9 

* Add RequireJS tests
* Add Webpack tests
* Add Karma drivers for each
* Add lint for new tests
* Update documentation
* Add code coverage and travis / coveralls support.

/cc @exogen @baer @aisapatino

### CODE REVIEW

**Comment by baer** [comment]
Testing JS is so complicated! This looks great - :ship:

File: README.md [hunk 1]
```diff
@@ -97,6 +105,20 @@ $ npm run server
...
+
+We run both Karma (client-side) and Selenium (functional) tests.
+
+The **Karma** tests are faster and more flexible, but slightly "off" from
+real-world use because of their execution environment. We use Karma tests to
+kick the tires on our AMD and CommonJS abstractions and little, one-off use
+case scenarios.
+
+The **Selenium** tests are slower and klunky, but they are the "real deal"
+executing `little-loader` in exactlty the same manner as would be used on a
```
Comment (exogen) [inline README.md]: typo, `exactlty`

Comment (ryan-roemer) [inline README.md]: You are exactlty correct.

**Comment by exogen** [comment]
LGTM :+1:

**Pull request merged**